### PR TITLE
Added notes for release 4.7.18

### PR DESCRIPTION
**Pull request by ctauchen**
Release notes for 4.7.18!

[Direct link here](https://deploy-preview-34046--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-7-release-notes.html#ocp-4-7-18)

### CODE REVIEW

File: release_notes/ocp-4-7-release-notes.adoc [hunk 1]
```diff
@@ -2542,3 +2542,48 @@ With this update, users can now collect the `virt_platform` metric. The `virt_pl
...
 
 To upgrade an existing {product-title} 4.7 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc#update-service-overview_updating-cluster-cli[Updating a cluster by using the CLI] for instructions.
+
+
+[id="ocp-4-7-18"]
+=== RHBA-2021:2502 - {product-title} 4.7.18 bug fix update
+
+Issued: 2021-06-28
+
+{product-title} release 4.7.18 is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHBA-2021:2502[RHBA-2021:2502] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2021:R2503[RHBA-2021:2503] advisory.
```
Comment (adellape) [inline release_notes/ocp-4-7-release-notes.adoc:2552]: Looks like an extra "R" in "R2503".

File: release_notes/ocp-4-7-release-notes.adoc [hunk 2]
```diff
@@ -2542,3 +2542,48 @@ With this update, users can now collect the `virt_platform` metric. The `virt_pl
...
+==== Features
+
+[id="ocp-4-7-18-builds-telemetry-metric"]
+===== New Telemetry metric for number of builds by strategy
+In this update, Telemetry includes a new `openshift:build_by_strategy:sum` gauge metric, which sends the number of builds by strategy type to the Telemeter Client. This metric gives site reliability engineers (SREs) and product managers visibility into the kinds of builds that run on {product-title} clusters. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1969963[*BZ#196963*])
+
+[id="ocp-4-7-18-bug-fixes"]
+==== Bug fixes
+
+* Previously, if you created a pipeline with parameters that had empty strings (`”`) as the default value, the fields in the {product-title} web console did not accept the empty strings. The current release fixes this issue. Now, `”` is supported as a valid default property within the parameters section. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1966275[*BZ#1966275*])
```
Comment (adellape) [inline release_notes/ocp-4-7-release-notes.adoc:2568]: Not super important, but:

In "within the parameters section", could/should `parameters` be marked up?

File: release_notes/ocp-4-7-release-notes.adoc [hunk 3]
```diff
@@ -2542,3 +2542,48 @@ With this update, users can now collect the `virt_platform` metric. The `virt_pl
...
+Space precluded documenting all of the container images for this release in the advisory. See the following article for notes on the container images in this release:
+
+link:https://access.redhat.com/solutions/6139672[{product-title} 4.7.18 container image list]
+
+[id="ocp-4-7-18-features"]
+==== Features
+
+[id="ocp-4-7-18-builds-telemetry-metric"]
+===== New Telemetry metric for number of builds by strategy
+In this update, Telemetry includes a new `openshift:build_by_strategy:sum` gauge metric, which sends the number of builds by strategy type to the Telemeter Client. This metric gives site reliability engineers (SREs) and product managers visibility into the kinds of builds that run on {product-title} clusters. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1969963[*BZ#196963*])
```
Comment (adellape) [inline release_notes/ocp-4-7-release-notes.adoc:2563]: BZ # doesn't match here. Looks like "196963" should be "1969963".

File: release_notes/ocp-4-7-release-notes.adoc [hunk 4]
```diff
@@ -2542,3 +2542,48 @@ With this update, users can now collect the `virt_platform` metric. The `virt_pl
...
+
+[id="ocp-4-7-18-bug-fixes"]
+==== Bug fixes
+
+* Previously, if you created a pipeline with parameters that had empty strings (`”`) as the default value, the fields in the {product-title} web console did not accept the empty strings. The current release fixes this issue. Now, `”` is supported as a valid default property within the parameters section. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1966275[*BZ#1966275*])
+
+* Previously, the Cluster Version Operator reported a `ClusterOperatorDegraded` alert when ClusterOperator resources were degraded for ten minutes, and this alert sometimes fired prematurely during installation as resources were still being created. This update changes the 10-minute period to 30 minutes, providing enough time for the installation to progress without premature `ClusterOperatorDegraded` alerts. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1957991[*BZ#1957991*])
```
Comment (adellape) [inline release_notes/ocp-4-7-release-notes.adoc:2570]: Markup `ClusterOperator`.

Comment (adellape) [reply release_notes/ocp-4-7-release-notes.adoc:2570]: s/ten/10/ 

to match the use of numerals in the next sentence.

IBM SG: "You can choose whether to use numerals or words for numbers that are less than 10."

File: release_notes/ocp-4-7-release-notes.adoc [hunk 5]
```diff
@@ -2542,3 +2542,48 @@ With this update, users can now collect the `virt_platform` metric. The `virt_pl
...
+
+* When installing to vSphere, the bootstrap machine sometimes did not correctly update the nameservers in the `/etc/resolv.conf` file. As a result, the bootstrap machine couldn't access the temporary control plane, and installations failed. This fix includes changes that makes finding the right line to update more reliable. With the bootstrap manager now able to access its temporary control plane, installations are able to succeed. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1967355[*BZ#1967355*])
```
Comment (adellape) [inline release_notes/ocp-4-7-release-notes.adoc:2572]: s/nameservers/name servers/

File: release_notes/ocp-4-7-release-notes.adoc [hunk 6]
```diff
@@ -2542,3 +2542,48 @@ With this update, users can now collect the `virt_platform` metric. The `virt_pl
...
+
+* Before this update, the Pipeline ServiceAccount did not use secrets created during the git import flow for private git repositories, causing these Pipelines to fail. This update fixes the problem by adding annotations to the secrets and to the Pipeline ServiceAccount. Pipelines for private git repositories now run correctly. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1970485[*BZ#1970485*])
```
Comment (adellape) [inline release_notes/ocp-4-7-release-notes.adoc:2574]: Use either:

```
`ServiceAcount` resource
```
or
```
service account
```

x2

Comment (adellape) [reply release_notes/ocp-4-7-release-notes.adoc:2574]: See [RH Supplementary Guide](https://redhat-documentation.github.io/supplementary-style-guide/#git) on usage for "Git" vs `git`.

File: release_notes/ocp-4-7-release-notes.adoc [hunk 7]
```diff
@@ -2542,3 +2542,48 @@ With this update, users can now collect the `virt_platform` metric. The `virt_pl
...
+
+* Previously, health checkers for Google Cloud Load Balancer left stale conntrack entries on cluster hosts, and this caused network interruptions to API server traffic that used the Google Cloud Load Balancer. This fix prevents the health check traffic from looping through the hosts, and this removed the network disruptions to the API server. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1949348[*BZ#1949348*])
```
Comment (adellape) [inline release_notes/ocp-4-7-release-notes.adoc:2576]: Markup `conntrack`?

File: release_notes/ocp-4-7-release-notes.adoc [hunk 8]
```diff
@@ -2542,3 +2542,48 @@ With this update, users can now collect the `virt_platform` metric. The `virt_pl
...
+
+* Previously, in the web console's Topology view, visualizations of Knative services displayed build status and repository information in the wrong location. This update adjusts which data are passed to the UI, and now only the correct information is displayed. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1954962[*BZ#1954962*])
```
Comment (adellape) [inline release_notes/ocp-4-7-release-notes.adoc:2578]: Markup:

```
*Topology*
```

File: release_notes/ocp-4-7-release-notes.adoc [hunk 9]
```diff
@@ -2542,3 +2542,48 @@ With this update, users can now collect the `virt_platform` metric. The `virt_pl
...
+
+* Previously, some `CredentialsRequest` custom resources with predefined Google Cloud Platform permissions were rejected when they were applied to a GCP project that did not use all the predefined permissions. With this fix, the Cloud Credentials Operator periodically fetches a list of allowed permissions to test, so now `CredentialsRequest` custom resources do not fail permissions checking. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1958983[*BZ#1958983*])
+
+* The fix for https://bugzilla.redhat.com/show_bug.cgi?id=1953097[*BZ#1953097*] enabled the CoreDNS bufsize plugin with a buffer size of 1232 bytes, even though some DNS resolvers are capable of handling a maximum of only 512 bytes. As a result, some DNS resolvers were unable to receive messages from the DNS Operator. This fix sets the buffer size to 512 bytes so those DNS resolvers can receive messages as expected. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1967766[*BZ#1967766*])
```
Comment (adellape) [inline release_notes/ocp-4-7-release-notes.adoc:2582]: Markup `bufsize`?

s/plugin/plug-in/

File: release_notes/ocp-4-7-release-notes.adoc [hunk 10]
```diff
@@ -2542,3 +2542,48 @@ With this update, users can now collect the `virt_platform` metric. The `virt_pl
...
+
+* Previously, root privileges were required to set extended file attributes while extracting RHEL images, which caused `oc image extract` to fail for normal users. With this update, `oc` checks user permissions and sets extended attributes only for the root user. `oc image extract` works correctly for both root and normal users. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1969928[*BZ#1969928*])
```
Comment (adellape) [inline release_notes/ocp-4-7-release-notes.adoc:2584]: s/`oc image extract`/the `oc image extract` command/

^ x2

s/`oc`/the OpenShift CLI (`oc`)/

s/RHEL/Red Hat Enterprise Linux (RHEL)/

File: release_notes/ocp-4-7-release-notes.adoc (hunk 2 above)
Comment (ctauchen) [reply release_notes/ocp-4-7-release-notes.adoc:2568]: Agreed! Changed!

File: release_notes/ocp-4-7-release-notes.adoc (hunk 3 above)
Comment (ctauchen) [reply release_notes/ocp-4-7-release-notes.adoc:2563]: Agreed! Changed!

File: release_notes/ocp-4-7-release-notes.adoc (hunk 4 above)
Comment (ctauchen) [reply release_notes/ocp-4-7-release-notes.adoc:2570]: Agreed! Changed!

File: release_notes/ocp-4-7-release-notes.adoc (hunk 6 above)
Comment (ctauchen) [reply release_notes/ocp-4-7-release-notes.adoc:2574]: Agreed! Changed!

File: release_notes/ocp-4-7-release-notes.adoc (hunk 8 above)
Comment (ctauchen) [reply release_notes/ocp-4-7-release-notes.adoc:2578]: Agreed! Changed!

File: release_notes/ocp-4-7-release-notes.adoc (hunk 9 above)
Comment (ctauchen) [reply release_notes/ocp-4-7-release-notes.adoc:2582]: Agreed! Changed!

File: release_notes/ocp-4-7-release-notes.adoc (hunk 10 above)
Comment (ctauchen) [reply release_notes/ocp-4-7-release-notes.adoc:2584]: Agreed! Changed!

File: release_notes/ocp-4-7-release-notes.adoc (hunk 5 above)
Comment (ctauchen) [reply release_notes/ocp-4-7-release-notes.adoc:2572]: Agreed! Changed!

File: release_notes/ocp-4-7-release-notes.adoc (hunk 7 above)
Comment (ctauchen) [reply release_notes/ocp-4-7-release-notes.adoc:2576]: Yes, I'd imagine so. Changed.

File: release_notes/ocp-4-7-release-notes.adoc (hunk 1 above)
Comment (ctauchen) [reply release_notes/ocp-4-7-release-notes.adoc:2552]: Good spot, thanks! Changed!

**Review by adellape — APPROVED** [review]

**Pull request merged**